### PR TITLE
feat: pass hono's context in `getLoadContext`

### LINKED ISSUE #23: Question: Why is it not possible to use `Context` from hono in `getLoadContext` ?
I reviewed the `createGetLoadContextArgs` function in src/remix.ts.
It creates the argument for `getLoadContext` using hono's `context`.
I believe it would be more practical if the argument of the `getLoadContext` function were directly hono's context. That way, we could leverage Hono's middleware in Remix.
However, this is not how it works in practice. I assume there must be a reason for this. Could you explain why?

### PR DESCRIPTION
**Pull request by yusukebe**
Fixes #23

### CODE REVIEW

File: examples/cloudflare-pages/app/routes/_index.tsx [hunk 1]
```diff
@@ -4,11 +4,12 @@ import { useLoaderData } from '@remix-run/react'
 export const loader = (args: LoaderFunctionArgs) => {
   const extra = args.context.extra
   const cloudflare = args.context.cloudflare
-  return { cloudflare, extra }
+  const myVarInVariables = args.context.hono.context.get('MY_VAR_IN_VARIABLES')
```
Comment (yusukebe) [inline examples/cloudflare-pages/app/routes/_index.tsx:7]: This `args.context.hono.context` seems verbose, but I don't have a good idea for the naming.

**Comment by ogadra** [comment]
Sorry for the late response.
This is exactly what I was looking for! Thank you!
I think it would be more considerate to update `examples/cloudflare-workers` .

**Comment by yusukebe** [comment]
@ogadra Thank you!

@predaytor What do you think of this? If okay, please review it.

**Comment by yusukebe** [comment]
@predaytor

Thanks. I also thought the Context Storage middleware could fit this use case. But the feature in this PR will be helpful!

**Comment by yusukebe** [comment]
Seems to be good! I'll merge this now. If you have any requests to change, please feel free to create your PR.

**Pull request merged**